man/crystal.1

```diff
@@ -139,7 +139,7 @@ Disable colored output.
 .It Fl -no-codegen
 Don't do code generation, just parse the file.
 .It Fl o
-Specify filename of output.
+Specify output path. If a directory, the filename is derived from the first source file (default: ./)
```

Comment:
Maybe:
```suggestion
Specify output path. If a directory, the filename is derived from the first source file (default is the current directory)
```

Same in `command.cr`.

Reply:
I'd think `./` should be clearly understandable as a reference to the current directory? Then this is just more words to say the same thing.
Do you have any concerns that `./` would not be understood correctly?

Reply:
I understand better the "current directory" words in a text than "./" which is first odd _and then_ I understand "oh, dot, the current directory".